### PR TITLE
Move quality histogram and keep order while renaming

### PR DESCRIPTION
**Pull request by KoalaQin**
Changes are: 
1. Move histograms to unfurl and add data_type to renaming; 
2. Keep order in `info` while renaming.

### CODE REVIEW

File: gnomad_qc/v4/create_release/validate_and_export_vcf.py [hunk 1]
```diff
@@ -484,7 +518,6 @@ def unfurl_nested_annotations(
 
 def make_info_expr(
     t: hl.Table,
-    hist_prefix: str = "",
```
Comment (mike-w-wilson) [inline gnomad_qc/v4/create_release/validate_and_export_vcf.py:487]: 495 should be dropped as well

**Review by mike-w-wilson — APPROVED** [review]
One thing, then this looks good to me! Where you put the hists makes more sense to me, not sure why they existed outside of the unfurl method in the first place 🤷 .  I wanted to review before Julia's because I found the hists werent being handled there so lets get this in to her branch and then I'll continue my review. One note, well need to reorder again in the VCF as we want certain fields to start in the info field. Not sure if that changes your approach here but I'm fine updating this part in the other PR.

File: gnomad_qc/v4/create_release/validate_and_export_vcf.py (hunk 1 above)
Comment (KoalaQin) [reply gnomad_qc/v4/create_release/validate_and_export_vcf.py:487]: L495? The CTT?

Comment (KoalaQin) [reply gnomad_qc/v4/create_release/validate_and_export_vcf.py:487]: I got what you mean, old L495, the docstring.

**Comment by KoalaQin** [comment]
Could you merge if this LGTY?

**Pull request merged**